packages/fxa-auth-server/test/remote/subscription_tests.js

```diff
@@ -235,19 +230,13 @@ describe('remote subscriptions:', function() {
       });
 
       it('should return active subscriptions', async () => {
-        let result = await client.getActiveSubscriptions(tokens[2]);
+        const result = await client.getActiveSubscriptions(tokens[2]);
         assert.isArray(result);
         assert.lengthOf(result, 1);
         assert.equal(result[0].createdAt, date * 1000);
         assert.equal(result[0].productId, PRODUCT_ID);
         assert.equal(result[0].uid, client.uid);
         assert.isNull(result[0].cancelledAt);
-
-        result = await client.account();
```

Comment:
Is there a separate test for this API call somewhere? Wondering why it's going away

Reply:
I can add that back. I had removed it because it was covered elsewhere and was inconsistently failing on my local.